docs/dev/building_the_image.md

```diff
@@ -0,0 +1,70 @@
+# Build the image
+
+There are multiple ways to build the image: cloud-based, local and local-containerized.
+
+## cloud-based
+
+We use GitHub Actions to build the image. The job is defined in [.github/workflow/build-image.yml](https://github.com/batchworksde/iksdp_desktop/actions/workflows/build-image.yml).
+
+As soon as you open a new development branch, commit, and push changes within the `debian-live` directory, the build process will run. It will take about 30 minutes to complete.
+
+When the build is successful, the image is copied to [http://iksdp.pfadfinderzaunten.org](http://iksdp.pfadfinderzaunten.org). The image will remain accessible there for 5 days.
+
+
+## local build 
+
+The local build must be run on a Debian-based distribution. We have tested Ubuntu 24.04, which is also used by the cloud-based build.
+
+run the local build:
+
+```bash
+git clone https://github.com/batchworksde/iksdp_desktop.git
+cd iksdp_desktop
+./debian-live/build.sh localBuild
```

Comment:
The `localBuild` parameter is not required because it is the default value if nothing has been defined.
https://github.com/batchworksde/iksdp_desktop/blob/a36c03a91d59dc9d1f2896aa4a534ed952216f85/debian-live/build.sh#L8

Reply:
updated.